### DISCUSSION TITLE
Is it possible to import by filter?

### DISCUSSION THREAD

**Discussion by aleemshaunali** [FlohGro-dev/Craftist#59] [Ideas]
The challenge I really want to address is to import Tomorrow's tasks into the respective Daily Notes. I can easily create a 'Tomorrow' filter but can't then import it. Thanks

**Comment by FlohGro-dev** [comment]
Sorry for the late response - I'm currently on a longer travel trip and don't have Internet access all the time.

That's a nice request and I already thought about it earlier. Feel free to open an issue in the repository. 

I must add that there really was no development in the API of Craft so I have no idea what the team will do with the API. This is why I mostly stopped to develop the extensions.
